core/src/loader/http/branding.cpp

```diff
@@ -65,7 +65,7 @@ QNetworkAccessManager::Operation GetBrandingOptionsLoader::operation() const {
 }
 
 GetBrandingCssLoader::GetBrandingCssLoader(ApiClient *apiClient)
-	: Jellyfin::Support::HttpLoader<QString, GetBrandingCssParams>(apiClient) {}
+	: Jellyfin::Support::HttpLoader<void, GetBrandingCssParams>(apiClient) {}
```

Comment:
I assume this endpoint has multiple results, it either returns 200 or 204, but the OpenAPI generator just picks one of the responses.

Reply:
Problem for future me: #41